foreshadow/tests/test_preprocessor.py

```diff
@@ -826,9 +829,6 @@ def test_preprocessor_serialize():
     out = proc.serialize()
 
     assert json.loads(json.dumps(truth)) == json.loads(json.dumps(out))
-    # with open(test_path, 'w') as outfile:
```

Comment:
I might leave the dumps comment in hear to make creating these files easy in the future explaining what it is.

Reply:
Given that it's only two lines and super easy to do I'd lean on not leaving them